ledger/sandbox/BUILD.bazel

```diff
@@ -279,3 +317,30 @@ client_server_test(
         "dont-run-on-darwin",
     ],
 ) if not is_windows else None
+
+client_server_test(
+    name = "conformance-test-wall-clock-h2database",
+    timeout = "long",
+    client = "//ledger/ledger-api-test-tool:ledger-api-test-tool",
+    client_args = [
+        "--all-tests",
+        "--exclude TimeIT",
+        "--timeout-scale-factor=10",
+    ],
+    data = [
+        "//ledger/test-common:dar-files",
+    ],
+    server = "//ledger/sandbox:sandbox-ephemeral-h2database",
```

Comment:
Same comment as above: we should be able to use `//ledger/sandbox:sandbox-binary` here.

Reply:
👍